C/Chafa/build_tarballs.jl

```diff
@@ -0,0 +1,31 @@
+using BinaryBuilder
+
+name = "Chafa"
+version = v"1.4.1"
+
+sources = [
+    ArchiveSource("https://hpjansson.org/chafa/releases/chafa-$(version).tar.xz",
+                  "46d34034f4c96d120e0639f87a26590427cc29e95fe5489e903a48ec96402ba3"),
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/chafa-*/
+./autogen.sh --prefix=${prefix} --build=${MACHTYPE} --host=${target}
```

Comment:
```suggestion
./autogen.sh \
    --prefix=${prefix} \
    --build=${MACHTYPE} \
    --host=${target} \
    ax_cv_gcc_check_x86_cpu_init=yes \
    ax_cv_gcc_check_x86_cpu_supports=yes
```

Reply:
Ok, now we've got FreeBSD in, I guess that's enough for you? :stuck_out_tongue:   

Jokes apart, my understanding is that this is true only when `${proc_family} == intel`